src/marks/delaunay.js

```diff
@@ -221,45 +221,45 @@ class Voronoi extends Mark {
         y: {value: y, scale: "y", optional: true},
         z: {value: z, optional: true}
       },
-      options,
+      initializer(options, function (data, facets, channels, scales, dimensions, context) {
+        let {x: X, y: Y, z: Z} = channels;
+        ({x: X, y: Y} = applyPosition(channels, scales, context));
+        Z = Z?.value;
+        const C = new Array(data.length);
+        const [cx, cy] = applyFrameAnchor(this, dimensions);
+        const xi = X ? (i) => X[i] : constant(cx);
+        const yi = Y ? (i) => Y[i] : constant(cy);
+        for (let facet of facets) {
+          if (X) facet = facet.filter((i) => isFinite(X[i]));
+          if (Y) facet = facet.filter((i) => isFinite(Y[i]));
```

Comment:
Should we use defined here instead of isFinite to match normal channel filtering? And also because isFinite(null) is true which is undesirable.

Reply:
yes